dotnetv3/IAM/README.md

```diff
@@ -1,99 +1,68 @@
+
+- [Attach a policy to a role](AttachRolePolicyExample/AttachRolePolicyExample/AttachRolePolicy.cs) (`AttachRolePolicyAsync`)
+- [Create an access key](CreateAccessKeyExample/CreateAccessKeyExample/CreateAccessKey.cs) (`CreateAccessKeyAsync`)
+- [Create a policy](CreatePolicyExample/CreatePolicy.cs) (`CreatePolicyAsync`)
+- [Create a role](CreateRoleExample/CreateRole.cs) (`CreateRoleAsync`)
+- [Create a user](CreateUserExample/CreateUserExample/CreateUser.cs) (`CreateUserAsync`)
+- [Delete an access key](DeleteAccessKeyExample/DeleteAccessKeyExample/DeleteAccessKey.cs) (`DeleteAccessKeyAsync`)
+- [Delete a role policy](DeleteRolePolicyExample/DeleteRolePolicyExample/DeleteRolePolicy.cs) (`DeleteRolePolicyAsync`)
+- [Delete a user](DeleteUserExample/DeleteUserExample/DeleteUser.cs) (`DeleteUserAsync`)
+- [Detach a policy from a role](DetachRolePolicyExample/DetachRolePolicyExample/DetachRolePolicy.cs) (`DetatchRolePolicyAsync`)
+- [Get the account password policy](GetAccountPasswordPolicyExample/GetAccountPasswordPolicy.cs) (`GetAccountPasswordPolicyAsync`)
+- [Get a policy](GetPolicyExample/GetPolicy.cs) (`GetPolicyAsync`)
+- [Get a role](GetRoleExample/GetRole.cs) (`GetRoleAsync`)
+- [List policies attached to a role](ListAttachedRolePoliciesExample/ListAttachedRolePolicies.cs) (`ListAttachedRolePoliciesAsync`)
+- [List groups](ListGroupsExample/ListGroups.cs) (`ListGroupsAsync`)
+- [List policies](ListPoliciesExample/ListPolicies.cs) (`ListPoliciesAsync`)
+- [List inline policies for a role](ListRolePoliciesExample/Program.cs) (`ListRolePoliciesAsync`)
+- [List roles](ListRolesExample/ListRoles.cs) (`ListRolesAsync`)
+- [List SAML providers](ListSAMLProvidersExample/ListSAMLProviders.cs) (`ListSAMLProvidersAsync`)
+- [List users](ListUsersExample/ListUsers.cs) (`ListUsersAsync`)
+
+### Scenarios
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+- [Create a user and assume a role](IAM_Basics_Scenario/IAM_Basics_Scenario/IAM_Basics.cs)
+
+## Run the examples
+
+### Prerequisites
+* To find prerequisites for running these examples, see the
+  [README](../README.md#Prerequisites) in the dotnetv3 folder.
+
+After the example compiles, you can run it from the command line. To
+do this, navigate to the folder that contains the .csproj file, and then
+issue the following command:
 
 ```
 dotnet run
 ```
 
 Alternatively, you can run the example from within your IDE.
 
-## Resources and documentation
-
-[AWS SDK for .NET Developer Guide](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/welcome.html)
-[AWS SDK for .NET API Reference Guide](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/index.html)
+## Additional resources
+* [Amazon Simple Email Service Developer Guide](https://docs.aws.amazon.com/ses/latest/dg/index.html)
+* [IAM API Reference](https://docs.aws.amazon.com/IAM/latest/APIReference/welcome.html)
```

Comment:
IAM API Reference --> AWS Identity and Access Management API Reference

Reply:
Fixed.